Src/libCZI/CziParse.h

```diff
@@ -33,8 +33,10 @@ class CCZIParse
         {
             kDimensionXyMustBePresent = 0,
             kDimensionOtherThanMMustHaveSizeOne,
+            kStoredDimensionOtherThanMMustHaveSizeOne,
             kDimensionMMustHaveSizeOneExceptForPyramidSubblocks,
             kDimensionMMustHaveSizeOne,
+            kblabla,
```

Comment:
?

Reply:
This is why it's draft, thank you for the catch!